R/package.R

```diff
@@ -37,15 +39,27 @@
 #' })
 #'
 #' prefix
+#'
+#' # modify tolower to fail if given a non-character
+#' tolower <- argufy(base::tolower, x = ~ is.character)
```

Comment:
Yeah, but that loads testthat. I think the `\dontrun` is actually fine. We should really have a way to have errors in examples. Examples are not test cases..... anyway, just leave it as it is.